src/nimblepkg/cli.nim

```diff
@@ -218,6 +218,9 @@ proc promptListInteractive(question: string, args: openarray[string]): string =
       of '\r':
         selected = true
         break
+      of '\3':
+        showCursor(stdout)
+        quit(1)
```

Comment:
You can use ``QuitFailure`` here instead. But what you should really do is raise some exception, ``NimbleError`` should be fine for this.

Reply:
k, this version throws a NimbleError instead.